doc/changes/latest.rst

```diff
@@ -10,6 +10,7 @@ NEW
 
 Fixes
 -----
+- Fix bug in :func:`nilearn.plotting.surf_plotting._plot_surf_matplotlib` that would make vertices transparent when saving in PDF or SVG format (:gh:`3860` by `Mathieu Dugré`_).
```

Comment:
The Sphinx cross-reference syntax won't work here because private functions (name starting with an underline) are not rendered in the docs.

```suggestion
- Fix bug in ``nilearn.plotting.surf_plotting._plot_surf_matplotlib`` that would make vertices transparent when saving in PDF or SVG format (:gh:`3860` by `Mathieu Dugré`_).
```